sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/CreateEcKeyOptions.java

```diff
@@ -101,32 +106,64 @@ public CreateEcKeyOptions setTags(Map<String, String> tags) {
     }
 
     /**
-     * Set the enabled value.
+     * Set a value that indicates if the key is enabled.
+     *
+     * @param enabled The enabled value to set.
      *
-     * @param enabled The enabled value to set
-     * @return the EcKeyCreateOptions object itself.
+     * @return The {@link CreateEcKeyOptions} object.
      */
     @Override
     public CreateEcKeyOptions setEnabled(Boolean enabled) {
         super.setEnabled(enabled);
+
+        return this;
+    }
+
+    /**
+     * Set a value that indicates if the private key can be exported.
+     *
+     * @param exportable The exportable value to set.
+     *
+     * @return The updated {@link CreateKeyOptions} object.
+     */
+    public CreateKeyOptions setExportable(Boolean exportable) {
```

Comment:
Shouldn't these methods return a `CreateEcKeyOptions`? I see this problem on a few methods.

Reply:
You are right, thanks for pointing this out.